packages/shared/src/hooks/referral/useJoinReferral.ts

```diff
@@ -47,6 +47,8 @@ export const useJoinReferral = (): void => {
         }
       }
     }
+
+    return null;
```

Comment:
Why is this return `null` needed?

Reply:
Getting an error for some reason. I think `useQuery` needs to have a return value always, probably running til the end of time if not.

https://app.circleci.com/jobs/github/dailydotdev/apps/27458